packages/components/src/entities/SampleTimelinePageBase.tsx

```diff
@@ -146,7 +146,7 @@ export const SampleTimelinePageBaseImpl: FC<OwnProps & InjectedQueryModels> = me
     };
 
     const auditDetailValueRenderer = (field: string, value: string, displayValue: any): any => {
-        if (field.toLowerCase() === 'sampleid') {
+        if (field.toLowerCase() === 'sample id') {
```

Comment:
so `field` here is the label? just to be safe, should this if statement check both (with and without the space)?

Reply:
It gets translated to have the space in it on the server side (specific to these timeline events), so I could add back the no-space version, but would rather have our tests fail (Selenium alerted me to this problem) if we remove that space again.